src/instructlab/model/test.py

```diff
@@ -30,10 +36,102 @@
     default="auto",
     show_default=True,
 )
-@utils.macos_requirement(echo_func=click.secho, exit_exception=click.exceptions.Exit)
+# for Linux:
+@click.option(
+    "-m",
+    "--model",
+    default=cfg.DEFAULT_MODEL,
+    show_default=True,
+    help="Base model name to test on Linux",
+)
+@click.option(
+    "-t",
+    "--test_file",
+    help="Test data file",
+)
+@click.option(
+    "--endpoint-url",
+    type=click.STRING,
+    help="Custom URL endpoint for OpenAI-compatible API. Defaults to the `ilab model serve` endpoint.",
```

Comment:
```suggestion
    help="Custom URL endpoint for OpenAI-compatible API. Defaults to the `ilab model serve` endpoint.",
    default="`ilab model serve` endpoint"
    show_default=True
```

Reply:
The value of `endpoint_url` is intended to be `None` by default.